### PR TITLE
Action status story

### PR DESCRIPTION
**Pull request by harshitabhatia26**
Story created for Action Status and Status Button.

### CODE REVIEW

File: lib/package.json [hunk 1]
```diff
@@ -54,6 +54,7 @@
     "@tiptap/pm": "2.1.13",
     "@tiptap/react": "2.1.13",
     "class-variance-authority": "^0.7.0",
+    "cva": "^0.0.0",
```
Comment (sudorock) [inline lib/package.json:57]: cva used to be an alias for `"class-variance-authority"` and I've updated the usage to be done from https://github.com/aktopia/elements/blob/master/lib/src/utils/style.ts

So this line should not be required. Are you facing issues without this?

File: storybook/src/stores/action/status.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { ActionStatus } from '@elements/logic/action';
+
+export const store = {
+  sub: {
+    'action.status/modal': { visible: true, 'action/id': '' },
```
Comment (sudorock) [inline storybook/src/stores/action/status.ts:5]: You can add a mock 'action/id' globally, say,
const actionId = 'action-1' and use actionId in the two places.

File: lib/package.json (hunk 1 above)
Comment (harshitabhatia26) [reply lib/package.json:57]: For me it is unable to detect it through class-varaince-authority unless i change the file imports to the same name or install cva instead.

Comment (harshitabhatia26) [reply lib/package.json:57]: For me it is unable to detect it through class-varaince-authority unless i change the file imports to the same name or install cva instead.

Comment (sudorock) [reply lib/package.json:57]: Try deleting your node_modules and installing again.

File: lib/src/compositions/issue/issue-header.tsx [hunk 3]
```diff
@@ -163,6 +166,46 @@ const ActionBar = () => {
   );
 };
 
+const IssueContextMenu = suspensify(() => {
```
Comment (sudorock) [inline lib/src/compositions/issue/issue-header.tsx:169]: Can you also update the issue-header story to reflect this change?

Comment (sudorock) [reply lib/src/compositions/issue/issue-header.tsx:169]: This way I can verify the changes in storybook

**Review by sudorock — APPROVED** [review]

**Pull request merged**